llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -434,6 +434,7 @@ def forward(
                 attn_bias=attn_bias,
                 attention_mask=attention_mask,
                 is_causal=self.is_causal,
+                output_attentions=output_attentions == True,
```

Comment:
```suggestion
                output_attentions=bool(output_attentions),
```
any reason this version doesn't work? the type is `Optional[bool]`, so should either be `None` or a `bool` and `bool(None)` is `False`